core/services/feeds/models.go

```diff
@@ -87,11 +90,24 @@ func NewChainType(s string) (ChainType, error) {
 		return ChainTypeEVM, nil
 	case "STARKNET":
 		return ChainTypeStarknet, nil
+	case "SOLANA":
+		return ChainTypeSolana, nil
+	case "APTOS":
+		return ChainTypeAptos, nil
```

Comment:
we are missing cosmos as well, but we can get this PR in first (not that we need it for aptos)